src/wizards/tokenSwapDealWizard/wizardManager.ts

```diff
@@ -248,6 +248,10 @@ export class WizardManager implements IRouteableComponent {
     // Getting the index of currently active stage route.
     // It is passed to the wizardService registerWizard method to register it with correct indexOfActive
     this.activeIndex = this.stages.findIndex(stage => stage.route.includes(stageRoute));
+
+    this.stages.forEach(stage => {
+      this.container.register(Registration.instance(`wizardSettings.${stage.route}`, {}));
+    });
```

Comment:
Could add a small comment, of what this is doing?

Reply:
added